### PR TITLE
Add instructions to run a local Airflow server

### PR DESCRIPTION
**Pull request by will-cromar**
# Description

See README diff

# Tests

Tested instructions locally with a WIP DAG.

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [x] I have performed a self-review of my code.
- [x] I have necessary comments in my code, particularly in hard-to-understand areas.
- [x] I have run one-shot tests and provided workload links above if applicable. 
- [x] I have made or will make corresponding changes to the doc if needed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,16 @@ Thank you for your interest in contributing to this project!
 
 Please review the [contribution guidelines](docs/contributing.md), and note that all contributions must adhere to the [code of conduct](docs/code-of-conduct.md).
 
+### Testing changes locally
```
Comment (RissyRan) [inline README.md:23]: Do you want to move this into this [contributing process step](https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/03188791e1548defe894f43aadb81f18df7120d4/docs/contributing.md)? So we have contributing guide together?

**Review by RissyRan — COMMENTED** [review]
Thanks Will! How will this feature look like? 

I ran this standalone before, and recall I need to do some setup. Is this the same for you? Could you provide one simple example for the run? I think this will be helpful for users as well.

File: README.md (hunk 1 above)
Comment (RissyRan) [reply README.md:23]: Or it's actually better if we move that all content from contributing page to the main page [README.md](https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/master/README.md)?

Comment (will-cromar) [reply README.md:23]: Good idea. I'll move this to the contributing doc.

**Comment by will-cromar** [comment]
> Thanks Will! How will this feature look like?
> 
> I ran this standalone before, and recall I need to do some setup. Is this the same for you? Could you provide one simple example for the run? I think this will be helpful for users as well.

I actually didn't have to do any other setup besides authenticating with GCP `gcloud auth login --update-adc` . `airflow standalone` does the one-time setup and prints a link to a local server. I didn't try all of the DAGs, so I don't expect them all to work. Some might be coupled to the composer environment, or people might not have the right permissions (e.g. to directly create QR in the project).

I'm contributing this as a quick hack that may or may not work, but we should expand it over time to work with more task types.

File: docs/contributing.md [hunk 2]
```diff
@@ -47,4 +47,15 @@ Run `bash scripts/format-codes.sh` to format your codes.
 
 All submissions, including submissions by project members, require review. We
 use [GitHub pull requests](https://docs.github.com/articles/about-pull-requests)
-for this purpose.
\ No newline at end of file
+for this purpose.
+
+### Testing changes locally
```
Comment (RissyRan) [inline docs/contributing.md:52]: Nit: `Testing changes locally` to `Testing Changes Locally` to match the format.

**Review by RissyRan — APPROVED** [review]
Approved with a nit.

**Pull request merged**